docs/root/intro/arch_overview/operations/hot_restart.rst

```diff
@@ -9,10 +9,8 @@ Envoy can fully reload itself (both code and configuration) without dropping exi
 during the :ref:`drain process <arch_overview_draining>`. The hot restart functionality has the
 following general architecture:
 
-* Statistics and some locks are kept in a shared memory region. This means that gauges will be
-  consistent across both processes as restart is taking place.
 * The two active processes communicate with each other over unix domain sockets using a basic RPC
-  protocol.
+  protocol. Statistics will be transported from the old process to the new process over the unix domain.
```

Comment:
We do not transport all stats - It would be good to qualify

Reply:
Do you mean `counters` and `gauges`?

Reply:
yes. For example, all gauges are not transported

Reply:
AFAIK, during hot restart, all counters and gauges will be **transported** to the new process, but after the new process sending the terminate request, the new process will clean up the contributions of most gauges transported from the old process except `hot_restart_generation` stat. Am I misunderstand?

Reply:
Some gauges are transported and some are not. Each gauge is marked as never import, import, etc.